modules/offlinemaps/src/main/java/com/sygic/maps/offlinemaps/loader/RegionHolder.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2020 Sygic a.s. All rights reserved.
+ *
+ * This project is licensed under the MIT License.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.sygic.maps.offlinemaps.loader
+
+import com.sygic.sdk.map.MapLoader
+
+class RegionHolder(val region: Region, status: MapLoader.MapStatus) : MapHolder(region.iso, status, 0) {
```

Comment:
neda sa urobit `data class` ?

Reply:
Tu sa to nedalo tak jednoducho kedze je to subclassa k `MapHolder`. Nakoniec som to cele zjednodusil tak ze tam pouzivam kompoziciu a `RegionHolder` ako aj `CountryHolder` maju `MapData`, kde su vsetky tieto data co boli dedene a tym padom to netreba robit ako subclassu a moze to byt `data class`.